actnow/petitions/serializers.py

```diff
@@ -6,7 +6,7 @@
 
 
 class SignatureSerializer(serializers.ModelSerializer):
-    signatory = UserSerializer(read_only=True)
+    signatory = serializers.HiddenField(default=serializers.CurrentUserDefault())
```

Comment:
What does this mean?

Reply:
the signature will automatically belong to the logged-in user.

Reply:
removes the need to send a user id while signing a petition.

Reply:
Yes but [docs](https://www.django-rest-framework.org/api-guide/validators/#advanced-field-defaults) says using `HiddenField` means the value will not be there in the output representation. This is what I'm what it actually means i.e. does it mean from signature output we won't be able to tell who signed it?

Reply:
Yes, with this, user data won't be returned. Right now, a user can only access their signatures so adding the user details initially looked like overkill.   
But. If we want the ability to display all signatures/petitions and the people who did the signing then I can look for an alternative way to achieve the above objective.

Reply:
I think the most common use of signatures is when someone is viewing a petition. These users are usually not the one who created the petition and most probably they don't even have an account at this point e.g. I can view this [petition](https://www.change.org/p/south-african-post-office-postofficesa-and-icasa-icasa-org-we-have-right-to-choose-who-delivers-our-small-parcels?source_location=petitions_browse) with signatures without login.

Now with this design, when we return a petition with signatures, will the info be visible or not?

Reply:
>Now with this design, when we return a petition with signatures, will the info be visible or not?

It won't. I'll revert to include the info

Reply:
Fixed.